cypress/integration/userCanSeeAnApplication.feature.js

```diff
@@ -1,9 +1,12 @@
-describe('user can see an app', () => {
+describe("interface", () => {
   beforeEach(() => {
-    cy.visit('/', {})
-  })
+    cy.intercept("GET", "**/api/products", {
+      fixture: "products.json",
+    }).as("getProducts");
```

Comment:
There is no need to store the intercept in an variable. Please remove that.

Reply:
By the way, is this test still passing?

Reply:
yes

Reply:
> There is no need to store the intercept in an variable. Please remove that.

changed